apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -116,46 +120,84 @@ class MenuBubbleUnwrapped extends Component {
       <div style={style} className={className}>
         {this.renderMenuItems()}
         {/* These elements are used to draw the 'tail' with CSS */}
-        <span style={tailBorderStyle}/>
-        <span style={tailFillStyle}/>
+        {!this.props.minimalStyle &&
```

Comment:
I think it'd be nice to have Mark/Poorva choose one style for these. As discussed, the carrot on/off is a reasonable flag, but it feels like we would want these to be the same anyways.